public/css/style.css

```diff
@@ -209,13 +212,13 @@ html body{
   color: #00b96b !important;
 }
 .fs1{
-  font-size: 0.9em;
+  font-size: 0.9em !important;
```

Comment:
yeah, its was created by said so that we can apply it where we need it

Reply:
I understand, We have agreed to stop using `!important` already

Reply:
If u don't need it don't use the class. However it doesn't touch other or override other part except you use the same class name and this is the importance of important, in this case my work will not be affected now is left for u to find out that the class name is used. Note: it is not use on a tag @Auwalms

Reply:
Yes @saidabdul80, I didnt see a good reason why he added !important since its not going to clash with any class. @softdiddy please remove the !important you added here.

Reply:
and also Note: it is used on a class with just one or two properties not class with many property which You might need to modify one of it to suit your work. it 💯 not affecting any one's work.

Reply:
The reason of use is to avoid override sir, let assume someone else in the team wrote a code that can override that property while he intend to use it somewhere else @Auwalms  please refer to the message before this and understand sir

Reply:
We will review and tell the person na. Its the reason why we said no !important